File: backend2/DataAccess/Common/Models/DaNameModel.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: 名称マスタ
//
// 作成日　　: 2024.07.10
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class DaNameModel
    {
        public string nmcd { get; set; } = string.Empty;   //名称コード
        public string nm { get; set; } = string.Empty;     //名称
        public DaNameModel(string nmcd, string nm)
        {
            this.nmcd = nmcd;
            this.nm = nm;
        }
    }
}

File: backend/BusinessService/Db/Common/Models/DaNameModel.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: 名称マスタ
'
' 作成日　　: 2024.07.10
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class DaNameModel
        Public Property nmcd As String = String.Empty   '名称コード
        Public Property nm As String = String.Empty     '名称
        Public Sub New(nmcd As String, nm As String)
            Me.nmcd = nmcd
            Me.nm = nm
        End Sub
    End Class
End Namespace
